### ISSUE TITLE
[Boss/Dungeon] Ambassador Hellmaw (Shadow Labyrinth)

### ISSUE THREAD

**Issue by Oganku** [WoWManiaUK/Redemption#4878]
**Links:** https://wow.gamepedia.com/Ambassador_Hellmaw
https://www.wowhead.com/npc=18794/cabal-ritualist
https://www.youtube.com/watch?v=9aLlWZdHa74

**What is Happening:** [Ambassador Hellmaw](https://wow.gamepedia.com/Ambassador_Hellmaw), the first boss of Shadow Labyrinth just patrols his room, and will not fight back if attacked, even until his death. Killing the [Cabal Ritualists](https://www.wowhead.com/npc=18794/cabal-ritualist) does trigger the boss and play his quote, upon which he then will fight back, but his pathing remains the same.

**What Should happen:** The boss should be "banished" in between the back 2 pillars of the room, and inactive until all channeling Cabal Ritualists have been defeated.

> After killing the last of the ritualists Hellmaw will path one time around the room. He will stop where he initially was standing before the last mob was killed, then patrol up and down the northern most hallway indefinitely.

Basically he should be un-attackable until the ritualists are killed. The trigger for him becoming hostile works, but the banishing/stationary part at first, doesn't.

[Video](https://www.youtube.com/watch?v=9aLlWZdHa74
) from above, although from cata, shows the behavior of the boss before and after killing the ritualists (around 6:50). You can even see pets attacking the banished boss, causing it to remain immune and stationary. Only after killing the last ritualist does he actually engage the party.
